src/js/rollup.config.es5.js

```diff
@@ -36,7 +36,7 @@ export default {
           {
             modules: false,
             targets: {
-              browsers: '> 1%'
+              browsers: '> 0.5%'
```

Comment:
This is the big problem that was breaking everything in IE

Reply:
interesting